docs/getting-started/index.md

```diff
@@ -21,6 +21,7 @@ you to use the [Contributors](contributors.md) guide to do a repository-based in
 
 - Access to a Unix-based operating system, such as Linux, MacOS or
   [Windows Subsystem for Linux](https://docs.microsoft.com/en-us/windows/wsl/)
+- `make` and `build-essential` installed
```

Comment:
Hm this is probably only true on linux, and possibly even just debian right? I might be wrong on that. On macos for example, you need to `xcode-select --install` instead.